parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnReaderImpl.java

```diff
@@ -523,7 +524,9 @@ private void readRepetitionAndDefinitionLevels() {
   private void checkRead() {
     if (isPageFullyConsumed()) {
       if (isFullyConsumed()) {
-        if (DEBUG) LOG.debug("end reached");
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
The call to debug will internally check the log level, right? I thought that it is only necessary to add a conditional on `isDebugEnabled` if the operation is expensive or if you're avoiding logging in a tight loop like you do in the writers. For writing a static debug string, is this needed?